static/app/components/replays/playerRelativeTime.tsx

```diff
@@ -12,9 +15,38 @@ const PlayerRelativeTime = ({relativeTime, timestamp}: Props) => {
     return <div />;
   }
 
-  return <Value>{showPlayerTime(timestamp, relativeTime)}</Value>;
+  return (
+    <Wrapper>
+      <Tooltip
+        title={<DateTime date={timestamp} />}
+        disabled={!timestamp}
+        skipWrapper
+        disableForVisualTest
+        underlineColor="gray300"
+        showUnderline
+      >
+        <Value>{showPlayerTime(timestamp, relativeTime)}</Value>
+      </Tooltip>
+    </Wrapper>
+  );
 };
 
+const Wrapper = styled('div')`
+  display: flex;
+  justify-content: center;
+  position: relative;
+  :before {
+    content: '';
+    display: block;
+    width: 1px;
+    top: 0;
+    bottom: 0;
+    left: 50%;
+    transform: translate(-50%);
+    position: absolute;
+  }
+`;
+
```

Comment:
Actually, is the Wrapper needed at all? The tooltip should position itself just fine.

Reply:
Yep, you were right, don't know why but seems to be in other places that are using a `<Wrapper>` for the Tooltip component, just realize doesn't need it anymore. Deleted.